index.js

```diff
@@ -6,6 +6,84 @@ class Subsume {
 	static parse(str, id) {
 		return (new Subsume(id)).parse(str);
 	}
+
+	static parseAll(str, ids) {
+		if (ids && typeof ids[Symbol.iterator] !== 'function') {
```

Comment:
I know I said an iterable, but I think it's better to accept an array. Strings can be iterables, which makes it super weird for the user if they incorrectly pass in a string.